MinecraftClient/ChatBots/MineCube.cs

```diff
@@ -0,0 +1,517 @@
+                Material.FletchingTable,
+                Material.Jukebox,
+                Material.Ladder,
+                Material.Lectern,
+                Material.Loom,
+                Material.NoteBlock,
+                Material.SmithingTable,
+                Material.TrappedChest,
+            });
+
+            // Search for keywords instead of every color
+            if (pickaxe.Contains(block) || block.ToString().Contains("Concrete") || block.ToString().Contains("GlazedTerracotta") ||
+                        (block.ToString().Contains("slab") &&
+                        !block.ToString().Contains("Oak") &&
+                        !block.ToString().Contains("Spruce") &&
+                        !block.ToString().Contains("Birch") &&
+                        !block.ToString().Contains("Jungle") &&
+                        !block.ToString().Contains("Arcacia") &&
+                        !block.ToString().Contains("Crimson") &&
+                        !block.ToString().Contains("Warped")))
+            {
+                return new ItemType[] { ItemType.NetheritePickaxe, ItemType.DiamondPickaxe, ItemType.IronPickaxe, ItemType.GoldenPickaxe, ItemType.StonePickaxe, ItemType.WoodenPickaxe};
+            }
+            else if (axe.Contains(block) ||
+                        //block.ToString().Contains("Fence") ||
+                        block.ToString().Contains("Banner") ||
+                        //block.ToString().Contains("Plank") ||
+                        //block.ToString().Contains("Sign") ||
+                        //block.ToString().Contains("Log") ||
+                        block.ToString().Contains("Hyphae") ||
+                        //block.ToString().Contains("Button") && block != Material.StoneButton ||
+                        //block.ToString().Contains("Door") && block != Material.IronDoor ||
+                        //block.ToString().Contains("PressurePlate") && block != Material.StonePressurePlate ||
+                        block.ToString().Contains("Oak") ||
+                        block.ToString().Contains("Spruce") ||
+                        block.ToString().Contains("Birch") ||
+                        block.ToString().Contains("Jungle") ||
+                        block.ToString().Contains("Arcacia") ||
+                        block.ToString().Contains("Crimson") ||
+                        block.ToString().Contains("Warped"))
+                 {
+                     return new ItemType[] { ItemType.NetheriteAxe, ItemType.DiamondAxe, ItemType.IronAxe, ItemType.GoldenAxe, ItemType.StoneAxe, ItemType.WoodenAxe };
+                 }
+            else if (shovel.Contains(block) || block.ToString().Contains("ConcretePowder"))
+            {
+                return new ItemType[] { ItemType.NetheriteShovel, ItemType.DiamondShovel, ItemType.IronShovel, ItemType.GoldenShovel, ItemType.StoneShovel, ItemType.WoodenShovel};
+            }
+          
+            return null;
```

Comment:
I think it would be better to return an empty array here instead of null to prevent null reference.